packages/stockflux-launcher/public/popupWindow.css

```diff
@@ -0,0 +1,31 @@
+body {
+    overflow: hidden;
+    padding: 4vh 4vw;
+}
+.popup{
+    color:var(--text-grey-lighter);    
+}
+
+.popup-message{
+    text-align: center;
+    font-size: var(--font-large);
+}
+
+.popup-options {
+    text-align: right;
+}
+
+button {
+    color:var(--text-teal-light);
+    border-color:var(--text-teal-light);
+}
+
+.modal-cancel {
+    color:var(--text-teal-light);
+    border-color:var(--text-teal-light);
+}
+
+.modal-confirm{
```

Comment:
This looks the same as the one above, could use `button, .modal-cancel, .modal-confirm {}` rather than three separate styles